hi-ml-azure/run_pytest.py

```diff
@@ -42,6 +47,12 @@ class RunPytestConfig(param.Parameterized):
         doc="The file or folder of tests that should be run. This value is used as the first argument to start "
         "pytest, so it can also be a specific test like 'my_test.py::any_test'",
     )
+    module: str = param.String(
```

Comment:
Name and description are slightly misleading. Maybe it should be called "coverage_module", because which tests we run is completely independent of what we collect coverage on.

Reply:
Also say that if this is empty, no coverage is collected.

Reply:
Let's cross that bridge when we get there, and keep this PR simple and merge it asap